ai-assistant-product/README.md

```diff
@@ -1 +1,671 @@
 # ai-assistant
+
+<todo>
+
+## Demo
+
+### Demo project
+
+In the **ai-assistant-demo** Ivy project, we created demo assistants which can help you understand and develop your own AI assistant more effeciency.
+
+> [!IMPORTANT]
+> This demo project will create Ivy users and roles and override the main variables of the AI Assistant. Therefore, it is HIGHLY RECOMMENDED to run it on an Ivy engine in demo mode to protect your data.
+>
+> If you choose to run this demo in a production environment, please ensure you back up all Ivy variables related to AI Assistant and restore them once the demo is complete.
+
+#### Complex Demo
+
+**Use case**
+
+User can plan a software project with AI Assistant. The AI Assistant enables users to efficiently plan a software project by organizing information into a structured format. It can:
+
+- Create the project and persist it in the database
+- Assist in finding and selecting suitable team members for the project
+- Help users prepare for the project kick-off meeting
+- Create and assign tasks to invite team members to the meeting
+
+**How to use**
+
+1. Run the process `startComplexDemo` to create test data and replace the original AI Assistant with the **Complex Demo Assistant**
+
+2. From **Portal**, open the AI Assistant app
+
+3. Now you can use the assistant to plan and create an software project by input the details of the project, and maybe number of members, or technologies.
+
+Example:
+
+`The XYZ Solutions Web Development Project aims to build a state-of-the-art web application that enhances user experience and supports the company's growth objectives. Mainly use basic web technologies such as HTML and CSS, the project will deliver a secure, scalable, and efficient platform that aligns with industry standards and best practices. We need 3 engineer, 1 web designer and 2 tester for it.`
+
+#### Eror Handling Demo
+
+**Use case**
+
+There are no data for meeting room. Therefore whenever user find for meeting room, AI Assitant will show error.
+This is an simple example how to help AI Assistant handle error properly.
+
+**How to use**
+
+1. Run the process `startErrorHandlingDemo` to create test data and replace the original AI Assistant with the **Error Handling Demo Assistant**
+
+2. From **Portal**, open the AI Assistant app
+
+3. Now you can run the demo by request to information of some room with **Error Handling Demo Assistant**.
+
+Example:
+
+`I want to find meeting room C`
+
```

Comment:
General: Shouldn't there be a general description how to train the AI assistant so these use cases do work ? What must be done to the assistant knows the rooms ? What must be trained so that the AI assistant can set up a project?